tests/lib/snaps/test-snapd-tools-core22/bin/sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cat <<EOM
+Launching a shell inside the default app confinement. Navigate to your
+app-specific directories with:
+
+  $ cd \$SNAP
+  $ cd \$SNAP_DATA
+  $ cd \$SNAP_USER_DATA
+
+EOM
+
+/bin/bash --norc -i
```

Comment:
Could this file (and many others) be a symbolic link?

Reply:
This is the same code that we use for the other test-snapd-tools snaps, so I would not change that once in this pr.
Also the snap is already published.
We could make that change in another pr and all the snaps should be updated in the store to be consistent

Reply:
OK!